test/tests/iotools/converters/adni_to_bids/test_adni_json.py

```diff
@@ -108,37 +118,60 @@ def _load_xml_from_template(
 
 
 def _write_xml_example(base_path, template_id="ADNI_123_S_4567", **kwargs):
-    """TODO."""
-    xml_content = _load_xml_from_template(**kwargs)
-    xml_file_path = base_path / f"{template_id}.xml"
+    suffix = kwargs.pop("suffix", None)
```

Comment:
Better use `def _write_xml_example(..., suffix: Optional[str] = None, **kwargs)` and avoid `kwargs.pop`. Explicit is better than implicit.

Reply:
I totally agree! Thanks! 👍